### ISSUE TITLE
Add LDAP auth

### ISSUE THREAD

**Issue by jeanfrancis** [etsy/opsweekly#38]

**Comment by gwinans** [comment]
I think the expectation here is that you'll will write your own auth scheme using the getUsername function in config.php

**Comment by lozzd** [comment]
That is correct... Maybe someone would enjoy writing an LDAP authentication function that others can share :)
